### ISSUE TITLE
Accessibility Screen Reader does not read hyphen or underscore in lyrics

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#19510]
### Issue type

UI bug

### Bug description

When either inputting lyrics or reviewing using NVDA it does not read out the hyphen between syllables or underscore used for a melisma or long note/tie and so a blind user has no way of knowing if these are present or being entered correctly.

### Steps to reproduce

Select a note to add a lyric to.  Press Ctrl L.  Type the lyric followed by a hyphen, it will automatically jump to the next note to add another lyric.  This time Add a lyric followed by an underscore.  If you arrow back through the lyrics just typed you will find it does not read the hyphen or dash and is also very hard to get it to consistently speak the letters to check spelling or to get the cursor in the correct place to edit.

Press Esc.  Pres Alt arrow to move back on to a note.  Arrow through the notes lyrics were just added to, the lyrics are read out after the note info but the dash and hyphen are not read.

Ideally these would be spoken by the screen reader when reviewing the score and when editing lyrics it would consistently read the character immediately to the right of the focus including hyphen & underscore as in any other text editing.

### MuseScore Version

4

### Regression

I don't know

### Operating system

Windows 10

**Comment by zacjansheski** [comment]
VoiceOver on MacOS does not give indication of hyphen/underscore either

**Comment by jmwmusic** [comment]
I see that hyphens and underscore in lyrics will now be able to be selected in 4.7.  Is it possible to also have them be navigated with Alt arrows so that they read with a screen reader.  Ideally navigated to as the next element directly after the syllable that they go after.
